dev_guide/new_app.adoc

```diff
@@ -91,12 +196,97 @@ $ oc new-app ruby-helloworld-sample
 ----
 ====
 
-When creating an application based on a stored link:templates.html[template],
-use the `-p|--param` to set parameter values defined by the template:
+To use a template in the file system directly, without first storing it in OpenShift, use
+the `-f|--file` argument or simply specify the file name as the argument to new-app:
 
+.Creating an application from a template in a file
+====
+----
+$ oc new-app -f examples/sample-app/application-template-stibuild.json
+----
+====
+
+==== Template parameters
+
+When creating an application based on a link:templates.html[template],
+use the `-p|--param` argument to set parameter values defined by the template:
+
+.Specifying template parameters with a template
+----
+$ oc new-app ruby-helloworld-sample -p ADMIN_USERNAME=<user>,ADMIN_PASSWORD=<pass>
```

Comment:
I really hate that we have new-app and process to do the same thing, and one takes -p to override params and one takes -v to override params.
[/rant]